### DISCUSSION TITLE
Shelly Temperature Addon & Tasmota KNX version

### DISCUSSION THREAD

**Discussion by Kolbi** [arendst/Tasmota#13795] [Ideas]
Hi guys,

I'm using an Shelly 1 + Shelly Temperature Addon and have connected 2 DS18B20.

I'm using the KNX version of Tasmota, but I only can send 1 temperature value, is it possible to increase this number to three?

Thanks in advance

**Comment by barbudor** [comment]
When looking at the code, it looks like the KNX interface supports only 1 temperature sensor per device
As far as I can see, If Tasmota was sending multiple Temperature to KNX, KNX wouldn't be able to know it's from different sensors.

**Reply by ascillato** [reply]
Yes, only one temperature source is supported by now, but it is doable since it is needed to assign a different KNX address to send data for each temperature sensor.

I'm adding this to my TO DO list for KNX. Thanks.

**Reply by Kolbi** [reply]
Thanks!

I'm new to Tasmota but I'll have a quick look into the code. Maybe I can find the right places and try to create a PR.
